index.html

```diff
@@ -241,6 +241,58 @@ <h2 class="about-title">About me</h2>
             </ul>
         </div>
     </section>
+    <section class="form-container">
+        <h3 class="form-heading">
+            I'm always interested in hearing about new projects, 
+            so if you'd like to chat please get in touch.
+        </h3>
+        <form action="https://formspree.io/f/xayvbrnw" method="post" class="form">
+            <ul class="form-inputs">
+                <li class="name-item">
+                    <input class="input" type="text" placeholder="Full name" name="full-name" maxlength="30" required>
+                    <div class="input-devider"></div>
+                </li>
+                <li class="email-item">
+                    <input class="input" type="email" placeholder="Email address" name="email" required>
+                    <div class="input-devider"></div>
+                </li>
+                <li class="comment-item">
+                    <textarea class="input-area" placeholder="Enter text here" name="comments" maxlength="500" required></textarea>
+                </li>
+            </ul>
+            <input class="submit-button" type="submit" value="Get in Touch">     
+        </form>
```

Comment:
- [ ] Please, consider that if you will place your contact form into `footer` it will be more professional and you will cover the semantic rules.
- [ ] You can use a simple form structure without the `ul`/`li` tags.
- [ ] Please, consider covering the design of the Figma template for your contact form:

> Figma template
<img width="212" alt="Screen Shot 2022-03-23 at 16 20 03" src="https://user-images.githubusercontent.com/49064106/159698805-36febdb2-7ebb-4e35-88df-68c8eb230107.png">

> You version
<img width="319" alt="Screen Shot 2022-03-23 at 16 26 03" src="https://user-images.githubusercontent.com/49064106/159698828-e75d1af0-aaa6-4774-b51c-44c0f6b47e90.png">